ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_instance_ip.schema.yml

```diff
@@ -6,6 +6,11 @@ keys:
   cvp_instance_ip:
     documentation_options:
       table: cloudvision-settings
+    deprecation:
+      warning: true
+      new_key: cvp_instance_ips
+      removed: true
```

Comment:
@Shivani-chourasiya / @carlbuchmann - we should not remove the key now - we should deprecate it only and remove it in 5.0.0

Reply:
okay, then we need to remove the test-case also

Reply:
Added molecule test